projboard/models.py

```diff
@@ -1,2 +1,65 @@
-# from django.db import models
-# from django.utils import timezone
+from django.db import models
+from django.utils import timezone
+
+
+class Subject(models.Model):
+    """
+    Subject model
+
+    name- A string, the subject name
+    """
+    name = models.CharField(max_length=100)
+
+
+class User(models.Model):
+    """
+    User model
+
+    email- User's Email
+    password- User's password
+    name- User's name
+    nickname- User's nickname
+    """
+    email = models.EmailField(max_length=150, unique=True)
+    password = models.CharField(max_length=70)
+    name = models.CharField(max_length=100)
+    nickname = models.CharField(max_length=100, unique=True)
+
+
+class Article(models.Model):
+    """
+    Article model
+
+    user_id- FK to User model
+    title- A string, the title of article
+    subject_id- FK to Subject model
+    content- A string, the content of article
+    date- A DateTime object, the date the article created
+    """
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    title = models.CharField(max_length=1000)
+    subject_id = models.ForeignKey(Subject, on_delete=models.SET_NULL, null=True)
```

Comment:
You should add , blank = True in order to make this field not required to allow null value

Reply:
in subject_id

Reply:
Fixed, look [here](https://github.com/redhat-beyond/Smarticle/blob/77494c0919ecb7ff5493139a518031e888ae5909/projboard/models.py)